.travis.yml

```diff
@@ -7,3 +7,9 @@ cache:
 script:
   - npm run build
   - npm run test
+
+deploy:
+  provider: firebase
+  token:
+    secure: "ACXqKxukQF5uhdFlkpAoQitNv9KG26oOUXouXKgZ8DbIF0yx/Nscs/LoKpll1aTpJku9qBWu8MbM0Wy742bWrPjWXJODX5ao3b4SunoJUjQt4RqXLInGJFiUa9jjBIHx+CQ7ru0ODqs00XLRW8tdmlfwN7VHF8fEI1jNkIXgd4NNpu6cQH7vajaDPrJTmq1TS9Wqp9tfgNme5XS2E90pfyBZewOqc8Go3B5s363QW0A1H3MW7xKDK5c8ICWgryg0Vu6oaBylGDhzPUQ1vpcsbOwTqf3j76Ok86mOb2ayO56+BaKL1h4JcobjIBXcHOWpfrs+n+zBlgn2ZLIk084H4v9cgoSbdG28jGpT9F/3AqQPcubkM2jJWeVIEucfRx1xgfahUH76H37Bo1V8e+uOgZOhdevxtptLEHYNRlRlFWYcF7zH3hFEVwu7G7nwbwj2X0UQr93F/MpN10M8Hkya0DtDWhuStRubTyg1nWuVMfJNyMvuoJGj50MEZo4PG1eOAjHxKozUzSxi3wyJI2+foxqpKcmSLtJnIqPWB5BAGhw+3tB9uWUSfsxdC3lRnBorR/D/VhYujJGV2yGTqewoIlOipqKg3zVupMjOcDgvs72zmdtA8WcUkiJ84pTOx97nWF8MwjCR3k/swTKvL4rQitlFWBS6xSA3cczXG186gtQ="
```

Comment:
Should this be private?

Reply:
No, this is the encrypted key. This is the way Travis instructs it done: https://docs.travis-ci.com/user/deployment/firebase/